libbitcoinkernel-sys/build.rs

```diff
@@ -52,12 +52,19 @@ fn main() {
         .status()
         .unwrap();
 
-    // Add the lib directory to the search path
-    let lib_dir = install_dir.join("lib");
+    // Check multiple possible lib directories
+    let possible_lib_dirs = vec![install_dir.join("lib"), install_dir.join("lib64")];
```

Comment:
Mmh, it is weird that you run into this at all. `cmake` by default puts everything into /lib on install, and we are installing into a local directory here. I don't think we should be making an exception here. Does the build pass in your environment if you instead add
```
        .arg("-DCMAKE_INSTALL_LIBDIR=lib")
```
To the cmake configuration above? That should override any interfering environment variables.

Reply:
Nice, that works! Also agree, this is much cleaner. I've update the first commit to use this approach.